packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -0,0 +1,258 @@
+  const { semanticColors, palette } = theme;
+  return {
+    [MS_HIGHCONTRAST_ACTIVE]: {
+      backgroundColor: 'Highlight',
+      borderColor: 'Highlight',
+      color: 'HighlightText',
+    },
+  };
+};
+
+export const getOptionStyles = memoizeFunction((
+  theme: ITheme,
+  customStylesForAllOptions?: Partial<IComboBoxOptionStyles>,
+  customOptionStylesForCurrentOption?: Partial<IComboBoxOptionStyles>,
+): IComboBoxOptionStyles => {
+
+  const { semanticColors, fonts, palette } = theme;
+
+  const ComboBoxoptionBackgroundSelected = semanticColors.menuItemBackgroundChecked;
+  const ComboBoxoptionBackgroundHovered = semanticColors.menuItemBackgroundHovered;
+
+  const optionStyles: IComboBoxOptionStyles = {
+    option: [
+      {
+        background: 'transparent',
+        boxSizing: 'border-box',
+        cursor: 'pointer',
+        display: 'block',
+        width: '100%',
+        height: 'auto',
+        minHeight: ComboBoxOptionHeight,
+        lineHeight: '20px',
+        padding: '5px 16px',
+        position: 'relative',
+        border: '1px solid transparent',
+        wordWrap: 'break-word',
+        overflowWrap: 'break-word',
+        textAlign: 'left',
+
+        [MS_HIGHCONTRAST_ACTIVE]: {
+          borderColor: 'Background'
+        }
+      },
+      getFocusStyle(theme),
+    ],
+    optionHovered: {
+      backgroundColor: ComboBoxoptionBackgroundHovered,
+      color: palette.black,
+      ...getListOptionHighContrastStyles(theme)
+    },
+    optionFocused: {
+      backgroundColor: ComboBoxoptionBackgroundHovered
+    },
+    optionActive: {
+      backgroundColor: ComboBoxoptionBackgroundHovered,
+      color: palette.black
+    },
+    optionSelected: [
+      {
+        backgroundColor: ComboBoxoptionBackgroundSelected,
+        color: palette.black,
+      },
+      getFocusStyle(theme),
+      getListOptionHighContrastStyles(theme)
+    ],
+    optionSelectedHovered: {
+      backgroundColor: ComboBoxoptionBackgroundSelected
+    },
+    optionDisabled: {
+      backgroundColor: palette.white,
+      color: palette.neutralTertiary,
+      cursor: 'default',
+      ' .ms-Button-flexContainer': {
+        justifyContent: 'flex-start'
+      }
+    },
+    optionText: {
+      overflow: 'hidden',
+      whiteSpace: 'nowrap',
+      textOverflow: 'ellipsis',
+      minWidth: '0px',
+      maxWidth: '100%',
+      wordWrap: 'break-word',
+      overflowWrap: 'break-word',
+      margin: '1px'
+    },
+    header: {
+      fontSize: fonts.medium,
+      fontWeight: FontWeights.semibold,
+      color: semanticColors.menuHeader,
+      background: 'none',
+      border: 'none',
+      height: ComboBoxOptionHeight,
+      lineHeight: ComboBoxOptionHeight,
+      cursor: 'default',
+      padding: '0px 16px',
+      userSelect: 'none',
+      textAlign: 'left'
+    },
+    divider: {
+      height: '1px',
+      backgroundColor: semanticColors.bodyDivider

```

Comment:
I know you're not doing style cleanup, but please change this to border instead for HC scenarios